oneview_redfish_toolkit/blueprints/computer_system.py

```diff
@@ -324,3 +330,25 @@ def _get_drives_from_sp(server_profile):
             jbods_drives.extend(drives_by_sas_logical_uuid)
 
     return jbods_drives
+
+
+def _get_server_profile_template_from_sp(sp_uri):
```

Comment:
What do you think about move this method to the [ComputerSystemService](https://github.com/HewlettPackard/oneview-redfish-toolkit/blob/master/oneview_redfish_toolkit/services/computer_system_service.py#L24)?